### ISSUE TITLE
CI: prevent stale-bot from closing "help-wanted" tickets (or tickets attached to projects)

### ISSUE THREAD

**Issue by thaJeztah** [docker/docs#19050]
### Is this a docs issue?

- [X] My issue is about the documentation content or website

### Type of issue

Other

### Description

The stale-bot automatically closes tickets after there's been no activity for some time. Some of these tickets were "acknowledged" (valid), but not yet being worked on.

While it's good to periodically clean up stale issues, we should also prevent tickets from being closed prematurely; (IMO) it's ok to have a backlog of tickets to work on; not all may have "top" priority, but having tickets to track things to improve is useful.

### Location

- https://github.com/docker/docs/issues/5798

### Suggestion

We should prevent tickets from being closed automatically if;

- a "help-wanted" label is set; these are tickets that may be "lower" priority, but could be "good first issues" for contributors
- a project is set (these may not be publicly visible, but we have some internal GitHub projects keeping backlogs for teams, such as "runtime-team" and "build-team"

**Comment by mat007** [comment]
Yes, we can do that. I’ll create a ticket!

**Comment by dvdksn** [comment]
thanks @mat007 and @thaJeztah

**Comment by thaJeztah** [comment]
Thank you! ❤️

**Comment by mat007** [comment]
This has been changed and deployed now. Do we want to remove the `lifecycle/frozen` label from e.g. #3282 to test it?

**Comment by dvdksn** [comment]
thanks @mat007! I removed `lifecycle/frozen` on #3282, let's see if it works :-)

**Comment by mat007** [comment]
I think it looks good, right?

**Comment by dvdksn** [comment]
Yeah, I'm not sure it the auto-close kicks in only 30 days from when the label was removed, but if it turns out there's something amiss we can reopen this issue :)

thanks again @mat007

**Comment by mat007** [comment]
> Yeah, I'm not sure it the auto-close kicks in only 30 days from when the label was removed

Ah that’s a good point, I never know if removing a label updates the issue or not. It probably does, indeed 😬

**Comment by dvdksn** [comment]
I reopened this issue because I noticed that stale-bot just added the frozen label to #3282.

**Comment by dvdksn** [comment]
**Verdict: INDETERMINATE** — This is a CI/GH Actions infrastructure issue (stale-bot configuration), not a docs content change; the fix was partially deployed in Feb 2024 but the issue was reopened because stale-bot was still mislabeling issues, and the configuration needs human investigation.

**Comment by thaJeztah** [comment]
OMG a bot took over @dvdksn 🤖🤖

**Comment by dvdksn** [comment]
Just doing my part in automating away my own job.
